README.md

```diff
@@ -6,8 +6,82 @@
 
 [![GitHub Workflow Status (Testing)](https://img.shields.io/github/workflow/status/TrueBrain/TrueWiki/Testing/master?label=master)](https://github.com/TrueBrain/TrueWiki/actions?query=workflow%3ATesting)
 
-TrueWiki is a wikitext server similar to [mediawiki](https://github.com/wikimedia/mediawiki) and/or [gollum](https://github.com/gollum/gollum).
-As default storage it uses git (instead of a database).
+TrueWiki is an opinionated wikitext server similar to [mediawiki](https://github.com/wikimedia/mediawiki) and/or [gollum](https://github.com/gollum/gollum).
+As default storage it uses GitHub (instead of a database).
+
+## Opinionated
+
+TrueWiki iterates over what mediawiki offers to bring a more modern approach to a wiki.
+These opinions are described below.
+
```

Comment:
Maybe more opinions:
* TrueWiki uses a folder-tree structure to organise pages (similar to Confluence or other commercial Wikis). There are no uncategorised/unlinked/free-floating pages as in MediaWiki.